RoboJackets/robocup-software
# Situation Analysis Module Minimum Viable Product

## Description
Pull request for getting the situation analysis module to the point where we can start developing plays around it. See issue for short term goals.

## Associated Issue
Issue #1314 

## Design Documents
[TODO](www.robojackets.org)

## Steps to test
### Test Case 1
1. Configure field to a given situation from the list and description 
2. Make sure the situation displays correctly 

Expected result: ???

##  Sub-issues 
- [ ] Fix pileup state transitions and sensitivity 
- [ ] Create Design Document
- [ ] Fix goalie clear state check
- [ ] Figure out basic information publishing scheme so plays can use info for scoring.
